__tests__/app.test.js

```diff
@@ -153,6 +154,99 @@ describe("/api/articles/:article_id/comments",()=>{
                     expect(msg).toBe("Invalid article_id")
                 })
     })
+
+
+
+    // Post
+    test("POST 201 and the created comment", () => {
+
+        const newComment = {
+            author: "butter_bridge",
+            body: "This is a test comment"
+        }
+
+        return request(app)
+        .post("/api/articles/1/comments")
+        .send(newComment)
+        .expect(201)
+        .then(({body})=>{
+            const {comment} = body
+            
+            const finalComment = {
+                author: "butter_bridge",
+                body: "This is a test comment",
+                article_id: 1
+            }
+            expect(comment).toMatchObject(finalComment)
+        })
+    })
+    test("POST 404 when given an valid but non-existent article_id",()=>{
+
+        const newComment = {
+            author: "butter_bridge",
+            body: "This is a test comment"
+        }
+
+        return request(app)
+            .post("/api/articles/100/comments")
+            .send(newComment)
+            .expect(404)
+                .then(({body})=>{
+                    const {msg} = body
+                    expect(msg).toBe("article_id does not exist")
+                })
+    })
+    test("POST 404 when given an valid but non-existent article_id",()=>{
```

Comment:
I have corrected this to a invalid article_id and a 400 err code

Reply:
I've got a test for valid and non-existent article_id and the one above is the test is for an invalid article_id so I've changed the description now